themes/spichka/resources/scripts/front-page.js

```diff
@@ -0,0 +1,55 @@
+import domReady from '@roots/sage/client/dom-ready';
+
+import Swiper from 'swiper';
+
+/**
+ * Application entrypoint
+ */
+domReady(async () => {
+  const sliders = document.querySelectorAll('.swiper');
+  const conf = {
+    direction: 'horizontal',
+    loop: true,
+    slidesPerView: 5,
+    // Responsive breakpoints
+    breakpoints: {
+      // when window width is >= 320px
+      320: {
+        slidesPerView: 1,
+      },
+      // when window width is >= 480px
+      480: {
+        slidesPerView: 2,
+      },
+      // when window width is >= 640px
+      640: {
+        slidesPerView: 5,
+      },
+    },
+  };
+
+  sliders.forEach((item) => {
+    new Swiper(item, conf);
```

Comment:
Please, check if new Swiper() initializes for all of the matches for given selector
And then you can remove this loop